app/helpers/my_modules_helper.rb

```diff
@@ -112,4 +112,63 @@ def my_module_due_status(my_module, datetime = DateTime.current)
       ''
     end
   end
+
```

Comment:
Please use more verbal method names as this helper is used in multiple controllers